src/test/ui/suggestions/into-convert.stderr

```diff
@@ -0,0 +1,44 @@
+error[E0308]: mismatched types
+  --> $DIR/into-convert.rs:6:16
+   |
+LL |     let x: A = B;
+   |            -   ^ expected struct `A`, found struct `B`
+   |            |
+   |            expected due to this
+   |
+help: call `into` on this expression to convert `B` into `A`
```

Comment:
It'd be nice to name the trait being used, as for newcomers this does seem magic otherwise. `call std::convert::Into on` instead `call `into` on`?